src/lib.rs

```diff
@@ -21,7 +21,9 @@ use std::{
     task::{Context, Poll},
 };
 use tokio::io::{AsyncRead, AsyncWrite, ReadBuf};
-use tonic::transport::{server::Connected, Certificate};
+use tonic::transport::server::Connected;
+mod certificate;
+pub use certificate::Certificate;
```

Comment:
SslConnectInfo.certs should ideally contain openssl x509 type. Using bytes as today requires users to reparse or create x509 type to access cert information. But this is orthogonal to this PR.

Reply:
I will merge this and work on top of it.
Thanks for the contributions.